utils/snappy-move-results/common.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Get OpenShift cluster details
+export cluster_name=$(oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}')
+export masters=$(oc get nodes -l node-role.kubernetes.io/master --no-headers=true | wc -l)
+export workers=$(oc get nodes -l node-role.kubernetes.io/worker --no-headers=true | wc -l)
+export infra=$(oc get nodes -l node-role.kubernetes.io/infra --no-headers=true | wc -l)
+export platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}')
+export cluster_version=$(oc get clusterversion | grep -o [0-9.]* | head -1)
```

Comment:
store the complete name of the build...for example "4.8.0-0.nightly-2021-05-10-092939"
there are cases we run tests on different nightly version of the same OpenShift version, it will help to identify them easily.

Or...you can also store data in such a way..
4.8 / 4.8.0-0.nightly-2021-05-10-092939 / .........

Reply:
ack